### PR TITLE
Deploy with graphics

### LINKED ISSUE #1277: Bug: deployment scripts don't bring in graphics from LFS
**Current behaviour/situation**
Deployment scripts do not pull in graphics assets that are stored with LFS. This has been a contingency since we noticed excessive LFS bandwidth quota usage.

**Expected state/behaviour**
It is possible to precisely bring in only those LFS assets that are needed for the software. Should not be brininging in any Inventor / CAD files.

**How do you know it is wrong**
GUI will fail to compile, desktop shortcuts will have no icons, desktop background will not be updated.

### PR DESCRIPTION
**Pull request by martukas**
Updates deployment scripts to only bring in graphics via LFS. Also updated desktop background and removed some unused icons.

Please follow instructions [this page](https://github.com/RespiraWorks/Ventilator/tree/issue_1277_deploy_with_graphics/software/utils/rpi_config)
to test deployment.

Some lines will need to be changed before merging to master.

Closes #1277

## Self-review checklist:

- [x] Self-review: looked through the `Files changed` tab, browsed repository in branch
- [x] Documentation updated - reflects changes in code, electrical or mechanical design
- [x] Documentation and graphics follow the [documentation style guide](https://github.com/RespiraWorks/Ventilator/wiki/Documentation-style-guide)
- [x] New content is linked, easily discoverable, does not require too many clicks
- [x] Follows other relevant parts of [contributor wiki](https://github.com/RespiraWorks/Ventilator/wiki)
- [x] PR has a descriptive name
- [x] Tagged relevant reviewers

### Software only:

- [x] Follows our [code style](https://github.com/RespiraWorks/Ventilator/wiki/Code-style)
- [x] Commented code, particularly in hard-to-understand areas
- [x] All tests pass - new and old, locally and on CI
- [x] No new warnings or static check failures
- [x] Tests that prove fix is effective or new feature works
- [x] Manual tests are explained, with instructions for reproducing them

### CODE REVIEW

**Comment by martukas** [comment]
failing to build controller on RasPI
```
admin@rw-ventilator:~/ventilator/software/utils/rpi_config $ ./update.sh 
Continuing with update...
Already up to date.
Building and deploying cycle controller...
Removing .pio
~/ventilator/software/controller ~/ventilator/software/controller
Deploying: stm32
Processing stm32 (platform: ststm32@15.4.1; board: custom_stm32)
--------------------------------------------------------------------------------
Tool Manager: Installing platformio/tool-dfuutil @ ~1.11.0
Error: Could not find the package with 'platformio/tool-dfuutil @ ~1.11.0' requirements for your system 'linux_aarch64'
```
@asmodai27 any idea what the problem might be?

**Comment by asmodai27** [comment]
> any idea what the problem might be?

TLDR: ststm32 platform relies on dfu-util for some of its functions (which I don't think we use), but this bring a dependency that is not yet available for linux arm64. We can either roll back to an old version of ststm32, or compile dfu-util ourselves on th pi and submit it to platformio team for integration to their registry.

This looks a lot like https://community.platformio.org/t/ststm32-could-not-find-the-package-with-platformio-tool-dfuutil-1-9-190708-requirements-for-your-system-linux-aarch64/24059

Albeit with a different version.
Also look at related [platformio issue](https://github.com/platformio/platform-ststm32/issues/436).
Looks like we have to either:
- build [dfu-util 1.11](http://dfu-util.sourceforge.net/) on the pi and submit it to platformio for integration to their package registry
- roll back to an old ststm32, which uses a version of dfu-util available for linux aarch64

I'll try to build the library myself, and see what to do from there.

**Comment by asmodai27** [comment]
Successfully built the package following instructions in the aforementioned thread. Asked platformio community how to share the package: https://community.platformio.org/t/ststm32-dfuutil-missing-for-linux-aarch64/28669

**Review by asmodai27 — COMMENTED** [review]
Not tested yet because I want to keep my pi in shape (in order to keep the `dfu-util` environment on there, at least until the issue is fixed), but this looks good to me.

**Comment by martukas** [comment]
Works now!

**Review by asmodai27 — APPROVED** [review]
LGTM
